### PR TITLE
Adds: Missing text size resource

### PR DESCRIPTION
**Pull request by AjeshRPai**
## Fixes 
This PR fixes the Build issue on the `trunk` due to missing dimen resource `@dimen/m3_sys_typescale_headline_large_text_size` after updating [Material to 1.9.0](https://github.com/wordpress-mobile/WordPress-Android/pull/18433),

##Explanation of changes 
This PR adds a dimen for the missing resource(`plans_purchase_success_title`) and updates the XML - `plans_purchase_success_fragment.xml` to reference it

## To test:
1. Check if the build is passing on CI 
2. Check if the code changes are correct 

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and running local tests

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

UI Changes testing checklist:
- [x] Portrait and landscape orientations.
- [x] Light and dark modes.
- [x] Fonts: Larger, smaller and bold text.
- [x] High contrast.
- [x] Talkback.
- [x] Languages with large words or with letters/accents not frequently used in English.
- [x] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [x] Large and small screen sizes. (Tablet and smaller phones)
- [x] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

File: WordPress/src/main/res/layout/plans_purchase_success_fragment.xml [hunk 1]
```diff
@@ -36,7 +36,7 @@
             android:text="@string/dashboard_card_plans_checkout_success_title"
             android:textAlignment="viewStart"
             android:textFontWeight="700"
-            android:textSize="@dimen/m3_sys_typescale_headline_large_text_size"
+            android:textSize="@dimen/plans_purchase_success_title"
```
Comment (ParaskP7) [inline WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: 👋 @AjeshRPai !

Question (❓): As per [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18433/commits/032c245e08642a4b5f2c9472a555b14a851504f0) commit that I replaced this `m3_sys_typescale_headline_large_text_size` dimension with the more generic `text_sz_extra_extra_large` dimension (`32sp`), that is, instead of creating a new custom such `plans_purchase_success_title` dimension, why did you chose creating a new custom one instead of reusing the generic one?

Comment (ParaskP7) [reply WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: 😄 @irfano just merged @ravishanker's #18539 PR, which actually fixes all that (see merge conflict)! 🤣

Comment (ParaskP7) [reply WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: I guess we can close this PR... 🤷 ✅

Comment (AjeshRPai) [reply WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: Hey @ParaskP7
>As per [this](https://github.com/wordpress-mobile/WordPress-Android/pull/18433/commits/032c245e08642a4b5f2c9472a555b14a851504f0) commit that I replaced this m3_sys_typescale_headline_large_text_size dimension with the more generic text_sz_extra_extra_large dimension (32sp), that is, instead of creating a new custom such plans_purchase_success_title dimension, why did you chose creating a new custom one instead of reusing the generic one?

Just to answer the question.  I don't think using the dimen value `text_sz_extra_extra_large` is a good practice, hence I added a new one. More thoughts here. https://androidp2.wordpress.com/2023/05/03/wp-android-rfc-colors-dimensions-guidelines/

Comment (ParaskP7) [reply WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: Great, thanks for explaining the reasoning behind that decision @AjeshRPai , let me think about it a bit and I'll come back to you on that. 🙇  😊 👍

Comment (ParaskP7) [reply WordPress/src/main/res/layout/plans_purchase_success_fragment.xml:39]: > ...@AjeshRPai , let me think about it a bit and I'll come back to you on that.

TL;DR: I agree (`paqN3M-TC-p2#comment-1139`). 💯
